docs/source/kedro_project_setup/configuration.md

```diff
@@ -8,7 +8,10 @@ We recommend that you keep all configuration files in the `conf` directory of a
 ```python
 CONF_SOURCE = "new_conf"
 ```
-
+You can also specify a source directory for the configuration files at run time using the `--conf-source` flag with the `kedro run` command as follows:
+```bash
+kedro run --conf-source = "new_conf"
```

Comment:
I think we might need to be a little bit clearer here - is `new_conf` a path or just folder location? Maybe just indicate what users could put there.

Reply:
Thanks Yetu! Updated in f6994bd76c335577c5cbd1dbfd87b3860cf5829a!